t/fcm-make/55-build-fortran-prog-unit-tail-comment.t

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#-------------------------------------------------------------------------------
+# (C) British Crown Copyright 2006-17 Met Office.
```

Comment:
2019?

Reply:
Yes. Happy new year * 2 will be done in bulk when we issue the new release.